### PR TITLE
Bump up near-primitives version to 0.16

### PR DESCRIPTION
**Pull request by silvestrpredko**

### CODE REVIEW

**Comment by silvestrpredko** [comment]
Workspaces have an ugly solution. But it's not critical for now. When it will be fixed we will back to using the same ```AccountId``` everywhere.

File: README.md [hunk 1]
```diff
@@ -1,32 +1,48 @@
...
-`near-client` is an RPC for [Near Protocol](https://near.org), written in Rust.
+`near-client` is an implementation of RPC [Near Protocol](https://near.org)
 
 ## A little bit detailed explatanion
-Since RPC call are just simple HTTP-requests to the NEAR's RPC server, it's possible for anybody to use it directly without using web3.js/SDK/etc. 
-Near Protocol provides a [documentation](https://docs.near.org/api/rpc/introduction) with how their RPC can be used. Sometimes it's not very convinient (e.g. you want to call those functions in your Rust code), so `near-client` could help you to avoid writing HTTP-requests on your own.
 
-Essentially, `near-client` uses [reqwest](https://github.com/seanmonstar/reqwest) library for building an HTTP client that will send requests over the network.
-For now, it provides you with common methods such as `view/deploy_contract/function_call/view_access_keys/etc`.
+The main purpose of the library is to be cross-platform. It could be compiled to any platform `web` through `wasm`, `x64`, `arm`, etc...
```
Comment (kstepanovdev) [inline README.md:8]: Probably, you'd like to say that wasm can be used in web apps disregarding the platform that had compiled it.

Comment (silvestrpredko) [reply README.md:8]: I would like to say that library is cross-platform. And could be compiled to any target.

**Review by kstepanovdev — APPROVED** [review]
Utterly exhaustive comments, good job. Nevertheless, those commended lines are a tech debt and must have been fixed as soon as possible.

**Pull request merged**